### PR TITLE
[#83] Add support for leaderlog on aarch64

### PR DESCRIPTION
**Pull request by tdiesler**
Add a naive Nix shell that can eventually be used to build cncli on all supported target platforms

[DO NOT MERGE]

This PR is merely here to document that there is work happening to support arm64. Perhaps this can get added to some dedicated feature branch that we can share/work on?

### CODE REVIEW

**Comment by tdiesler** [comment]
Currently, I see ...

```
  Downloaded 162 crates (17.6 MB) in 6.42s (largest was `gmp-mpfr-sys` at 5.4 MB)
   Compiling autocfg v1.0.1
   Compiling proc-macro2 v1.0.27
   Compiling unicode-xid v0.2.2
   Compiling syn v1.0.73
   Compiling libc v0.2.97
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-Wl,--as-needed" "-L" "/home/pi/.rustup/toolchains/stable-aarch64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-gnu/lib"
...
-unknown-linux-gnu/lib/libcore-d7926f131590f689.rlib" "-Wl,--end-group" "/home/pi/.rustup/toolchains/stable-aarch64-unknown-linux-gnu/lib/rustlib/aarch64-unknown-linux-gnu/lib/libcompiler_builtins-8e0dea81c2814a1a.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lutil" "-lrt" "-lpthread" "-lm" "-ldl" "-lc"
  = note: impure path `/home/pi/git/nessus-cardano/nix/cncli/target/cncli/target/release/build/syn-8064e11de6573109/build_script_build-8064e11de6573109' used in link
          collect2: error: ld returned 1 exit status
error: aborting due to previous error
error: could not compile `syn`
To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: linking with `cc` failed: exit status: 1
```

It fails with similar errors on all three target platforms. This is likely because of some dependency issue that I couldn't translate from the documented deps to Nix packages

**Comment by AndrewWestberg** [comment]
@disassembler ☝️

**Comment by tdiesler** [comment]
These build instructions in the main README can now be merged, if you like.
I also have the `cncli-2.1.1-aarch64-linux` binary that could be attached to the release. 
Although you might want to build that yourself.

Ideally, we would have the aarch64 binary as part of every release going forward.
If there is anything else, you'd like me to do just ping (e.g. tomdx on discord)
[ASTOR]

File: README.md [hunk 1]
```diff
@@ -12,6 +12,63 @@ To install CNCLI using either the release binaries or compiling the Rust code, o
 
 For a list of CNCLI commands and related usage examples, please refer to the [usage guide](USAGE.md).
 
+## Building
```
Comment (AndrewWestberg) [inline README.md:15]: Can you add these instructions instead to INSTALL.md instead of README.md?

Comment (tdiesler) [reply README.md:15]: sure

Comment (AndrewWestberg) [reply README.md]: Add yourself and your pool's website to the contributors list in README.md and then it'll be ready to merge.

**Comment by tdiesler** [comment]
Wow, thanks for the flowers ... and what are we going to do about those binaries. Are you taking care of this for the next release? If you have a [RasberryPi4 it would do](https://tdiesler.gitbook.io/cardano/v/iohk/plain-docker/raspberrypi-4), if not AWS also has aarch64 machines.

**Review by AndrewWestberg — APPROVED** [review]

**Pull request merged**